nltk/chunk/regexp.py

```diff
@@ -829,7 +829,7 @@ def __repr__(self):
 # this should probably be made more strict than it is -- e.g., it
 # currently accepts 'foo'.
 CHUNK_TAG_PATTERN = re.compile(r'^((%s|<%s>)*)$' %
-                                ('[^<>]+',
+                                ('([^\{\}<>]|\{\d+,?\d*\}|\{\d*,?\d+\})+',
```

Comment:
@chmeyer: I think there's a readability issue here. The case where a comma is omitted requires us to realise the `\d+\d*` is harmless. I would rather see the cases `{m}`, `{m,}`, `{,n}`, and `{m,n}` laid out more distinctly. E.g. as `{\d+,?}|{\d*,\d+}`. What do you think.

(NB I didn't escape the braces here, for readability)